comms/dht/src/outbound/broadcast.rs

```diff
@@ -306,6 +318,9 @@ where S: Service<DhtOutboundMessage, Response = (), Error = PipelineError>
                     }
                 }
 
+                // Todo, move config value in here for expires timestamp on messages
+                let expires = Utc::now() + Duration::seconds(self.message_validity_window);
```

Comment:
Convert `message_validity_window` (now a `std::time::Duration`) to a `chrono::Duration`

```suggestion
                let expires = Utc::now() + chrono::Duration::seconds(self.message_validity_window.as_secs());
```

For code clarity, I suggest removing the `use chrono::Duration` import, as that may be confusing in future with the canonical `std::time::Duration` and always using the unambiguous `chrono::Duration`

Reply:
Done.